<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>    <!-- Metadata, OpenGraph and Schema.org -->
    
    <!-- Website verification -->
    <meta name="google-site-verification" content="4uEvBxHywCbkVGzNxsKto3eDby-sB1z-z9JX4OW0zeI" /><meta name="msvalidate.01" content="E00E5A3C2D1B0E80FB4DF7738F7FFF3D" />

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Song Group | publications</title>
    <meta name="author" content="Chen  SONG" />
    <meta name="description" content="publications in reversed chronological order. generated by jekyll-scholar." />
    <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website" />


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous" />

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="none" id="highlight_theme_light" />

    <!-- Styles -->
    
    <link rel="shortcut icon" href="/assets/img/logo.png"/>
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://www.songgroup.org/publications/">
    
    <!-- Dark Mode -->
    
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark" />

    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav sticky-bottom-footer">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="https://www.songgroup.org//">Song Group</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/">about</a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item ">
                <a class="nav-link" href="/people/">people</a>
              </li>
              <li class="nav-item active">
                <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/resources/">resources</a>
              </li>

              <!-- Toogle theme mode -->
              <div class="toggle-container">
                <a id="light-toggle">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </a>
              </div>
            </ul>
          </div>
        </div>
      </nav>
    </header>


    <!-- Content -->
    <div class="container mt-5">
      <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">publications</h1>
            <p class="post-description">publications in reversed chronological order. generated by jekyll-scholar.</p>
          </header>

          <article>
            <!-- _pages/publications.md -->
<div class="publications">
  <h2 class="year">2023</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="he2023" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Knowledge-Guided Data Mining on the Standardized Architecture of NRPS: Subtypes, Novel Motifs, and Sequence Entanglements</div>
          <!-- Author -->
          <div class="author">He, Ruolin; Zhang, Jinyu; Shao, Yuanzhe; Gu, Shaohua; Song, Chen; Qian, Long; Yin, Wen-Bing; and Li, Zhiyuan
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>PLOS Computational Biology</em> May 2023
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.1371/journal.pcbi.1011100" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Non-ribosomal peptide synthetase (NRPS) is a diverse family of biosynthetic enzymes for the assembly of bioactive peptides. Despite advances in microbial sequencing, the lack of a consistent standard for annotating NRPS domains and modules has made data-driven discoveries challenging. To address this, we introduced a standardized architecture for NRPS, by using known conserved motifs to partition typical domains. This motif-and-intermotif standardization allowed for systematic evaluations of sequence properties from a large number of NRPS pathways, resulting in the most comprehensive cross-kingdom C domain subtype classifications to date, as well as the discovery and experimental validation of novel conserved motifs with functional significance. Furthermore, our coevolution analysis revealed important barriers associated with re-engineering NRPSs and uncovered the entanglement between phylogeny and substrate specificity in NRPS sequences. Our findings provide a comprehensive and statistically insightful analysis of NRPS sequences, opening avenues for future data-driven discoveries.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">he2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Knowledge-Guided Data Mining on the Standardized Architecture of {{NRPS}}: {{Subtypes}}, Novel Motifs, and Sequence Entanglements}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{He, Ruolin and Zhang, Jinyu and Shao, Yuanzhe and Gu, Shaohua and Song, Chen and Qian, Long and Yin, Wen-Bing and Li, Zhiyuan}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">may</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{false}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLOS Computational Biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{19}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e1011100}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1553-7358}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1371/journal.pcbi.1011100}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pcbi.1011100}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="schackert_mechanism_2023" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Mechanism of Calcium Permeation in a Glutamate Receptor Ion Channel</div>
          <!-- Author -->
          <div class="author">Schackert, Florian Karl; Biedermann, Johann; Abdolvand, Saeid; Minniberger, Sonja; Song, Chen; Plested, Andrew J. R.; Carloni, Paolo; and Sun, Han
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Journal of Chemical Information and Modeling</em> Feb 2023
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.1021/acs.jcim.2c01494" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid receptors (AMPARs) are neurotransmitter-activated cation channels ubiquitously expressed in vertebrate brains. The regulation of calcium flux through the channel pore by RNA-editing is linked to synaptic plasticity while excessive calcium influx poses a risk for neurodegeneration. Unfortunately, the molecular mechanisms underlying this key process are mostly unknown. Here, we investigated calcium conduction in calcium-permeable AMPAR using Molecular Dynamics (MD) simulations with recently introduced multisite force-field parameters for Ca2+. Our calculations are consistent with experiment and explain the distinct calcium permeability in different RNA-edited forms of GluA2. For one of the identified metal binding sites, multiscale Quantum Mechanics/Molecular Mechanics (QM/MM) simulations further validated the results from MD and revealed small but reproducible charge transfer between the metal ion and its first solvation shell. In addition, the ion occupancy derived from MD simulations independently reproduced the Ca2+ binding profile in an X-ray structure of an NaK channel mimicking the AMPAR selectivity filter. This integrated study comprising X-ray crystallography, multisite MD, and multiscale QM/MM simulations provides unprecedented insights into Ca2+ permeation mechanisms in AMPARs, and paves the way for studying other biological processes in which Ca2+ plays a pivotal role.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">schackert_mechanism_2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Mechanism of {Calcium} {Permeation} in a {Glutamate} {Receptor} {Ion} {Channel}}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{false}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{63}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1549-9596}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1021/acs.jcim.2c01494}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1021/acs.jcim.2c01494}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Chemical Information and Modeling}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Schackert, Florian Karl and Biedermann, Johann and Abdolvand, Saeid and Minniberger, Sonja and Song, Chen and Plested, Andrew J. R. and Carloni, Paolo and Sun, Han}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1293--1300}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">ProtRAP</abbr></div>

        <!-- Entry bib key -->
        <div id="kang2023a" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">ProtRAP: Predicting Lipid Accessibility Together with Solvent Accessibility of Proteins in One Run</div>
          <!-- Author -->
          <div class="author">Kang, Kai; Wang, Lei; and Song, Chen
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Journal of Chemical Information and Modeling</em> Jan 2023
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.1021/acs.jcim.2c01235" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Solvent accessibility has been extensively used to characterize and predict the chemical properties of the surface residues of soluble proteins. However, there is not yet a widely accepted quantity of the same dimension for the study of lipid-accessible residues of membrane proteins. In this study, we propose that lipid accessibility, defined in a similar way to solvent accessibility, can be used to characterize the lipid-accessible residues of membrane proteins. Moreover, we developed a deep learning-based method, ProtRAP (Protein Relative Accessibility Predictor), to predict the relative lipid accessibility and relative solvent accessibility of residues from a given protein sequence, which can infer which residues are likely accessible to lipids, accessible to solvent, or buried in the protein interior in one run.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">kang2023a</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{{ProtRAP}}: {{Predicting Lipid Accessibility Together}} with {{Solvent Accessibility}} of {{Proteins}} in {{One Run}}}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{{{ProtRAP}}}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{{ProtRAP}}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1021/acs.jcim.2c01235}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kang, Kai and Wang, Lei and Song, Chen}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Chemical Information and Modeling}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{{American Chemical Society}}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1549-9596}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1021/acs.jcim.2c01235}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">SecA</abbr></div>

        <!-- Entry bib key -->
        <div id="dong2023" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Structural Basis of SecA-mediated Protein Translocation</div>
          <!-- Author -->
          <div class="author">Dong, Linlin; Yang, Song; Chen, Jingxia; Wu, Xiaofei; Sun, Dongjie; Song, Chen; and Li, Long
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Proc. Natl. Acad. Sci. U.S.A.</em> Jan 2023
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.pnas.org/doi/10.1073/pnas.2208070120" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Secretory proteins are cotranslationally or posttranslationally translocated across lipid membranes via a protein-conducting channel named SecY in prokaryotes and Sec61 in eukaryotes. The vast majority of secretory proteins in bacteria are driven through the channel posttranslationally by SecA, a highly conserved ATPase. How a polypeptide chain is moved by SecA through the SecY channel is poorly understood. Here, we report electron cryomicroscopy structures of the active SecA-SecY translocon with a polypeptide substrate. The substrate is captured in different translocation states when clamped by SecA with different nucleotides. Upon binding of an ATP analog, SecA undergoes global conformational changes to push the polypeptide substrate toward the channel in a way similar to how the RecA-like helicases translocate their nucleic acid substrates. The movements of the polypeptide substrates in the SecA-SecY translocon share a similar structural basis to those in the ribosome-SecY complex during cotranslational translocation.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dong2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Structural Basis of {{SecA-mediated}} Protein Translocation}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dong, Linlin and Yang, Song and Chen, Jingxia and Wu, Xiaofei and Sun, Dongjie and Song, Chen and Li, Long}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proc. Natl. Acad. Sci. U.S.A.}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{120}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e2208070120}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1073/pnas.2208070120}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{SecA}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.pnas.org/doi/10.1073/pnas.2208070120}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2022</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="liu2022a" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Asymmetrical Calcium Ions Induced Stress and Remodeling in Lipid Bilayer Membranes</div>
          <!-- Author -->
          <div class="author">Liu, Chang; Zhong, Qi; Kang, Kai; Ma, Rui; and Song, Chen
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>ChemRxiv</em> Nov 2022
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.26434/chemrxiv-2022-24qv4" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Ca2+ ions play crucial roles in regulating many chemical and biological processes, but its impact on lipid bilayer membranes remains elusive, especially when the impacts on the two leaflets are asymmetrical. Using a recently developed multisite Ca2+ model, we performed molecular dynamics simulations to study the impact of Ca2+ on the properties of membranes composed of POPC and POPS, and observed that both the structure and fluidity of the membranes were significantly affected. Particularly, we examined the influence of asymmetrically distributed Ca2+ on asymmetric lipid bilayers, and found that imbalanced stress in the two leaflets were generated, with the negatively charged leaflet on the Ca2+ rich side becoming more condensed, which in turn induced membrane curvature that bent the membrane away from the Ca2+ rich side. We employed continuum mechanics to study the large-scale deformations of the membrane and found that membranes can develop into locally pearl-shaped or globally oblate, depending on the specific Ca2+ distributions. These results provide new insights to the underlying mechanism of many biological phenomena involving Ca2+-membrane interactions, and may lead to new methods for manipulating membrane curvature of vesicles in biological, chemical, and nano systems.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">liu2022a</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Asymmetrical Calcium Ions Induced Stress and Remodeling in Lipid Bilayer Membranes}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liu, Chang and Zhong, Qi and Kang, Kai and Ma, Rui and Song, Chen}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{ChemRxiv}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.26434/chemrxiv-2022-24qv4}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.26434/chemrxiv-2022-24qv4}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">TRPV</abbr></div>

        <!-- Entry bib key -->
        <div id="liu2022_TRPV" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Calcium Binding and Permeation in TRPV Channels: Insights from Molecular Dynamics Simulations</div>
          <!-- Author -->
          <div class="author">Liu, Chunhong; and Song, Chen
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>bioRxiv</em> Sep 2022
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.biorxiv.org/content/10.1101/2022.09.07.506889v1" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Some calcium channels selectively permeate Ca2+, despite the high concentration of monovalent ions in the surrounding environment, which is essential for many physiological processes. Without atomistic and dynamical ion permeation details, the underlying mechanism of Ca2+ selectivity has long been an intensively studied, yet controversial, topic. This study takes advantage of the homologous Ca2+-selective TRPV6 and non-selective TRPV1 and utilizes the recently solved open-state structures and a newly developed multi-site calcium model to investigate the ion binding and permeation features in TRPV channels by molecular dynamics simulations. Our results revealed that the open-state TRPV6 and TRPV1 show distinct ion-binding patterns in the selectivity filter, which lead to different ion permeation features. Two Ca2+ ions simultaneously bind to the selectivity filter of TRPV6 compared with only one Ca2+ in case of TRPV1. Multiple Ca2+ binding at the selectivity filter of TRPV6 permeated in a concerted manner, which could efficiently block the permeation of Na+. Cations of various valences differentiate between the binding sites at the entrance of the selectivity filter in TRPV6. Ca2+ preferentially binds to the central site with a higher probability of permeation, repelling Na+ to a peripheral site. Therefore, we believe that ion binding competition at the selectivity filter of calcium channels, including the binding strength and number of binding sites, determines Ca2+ selectivity under physiological conditions. Additionally, our results showed that pore helix flexibility and the cytosolic domain of TRPV channels regulate ion permeability.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">liu2022_TRPV</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Calcium Binding and Permeation in {{TRPV}} Channels: Insights from Molecular Dynamics Simulations}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Calcium Binding and Permeation in {{TRPV}} Channels}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liu, Chunhong and Song, Chen}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{TRPV}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">sep</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2022.09.07.506889}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{{bioRxiv}}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.biorxiv.org/content/10.1101/2022.09.07.506889v1}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/2022.09.07.506889}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">AltConf</abbr></div>

        <!-- Entry bib key -->
        <div id="li_exploring_2022" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Exploring the alternative conformation of a known protein structure based on contact map prediction</div>
          <!-- Author -->
          <div class="author">Li, Jiaxuan; Wang, Lei; Zhu, Zefeng; and Song, Chen
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>bioRxiv</em> Jun 2022
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.biorxiv.org/content/10.1101/2022.06.07.495232v1" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The rapid development of deep learning-based methods has considerably advanced the field of protein structure prediction. The accuracy of predicting the 3D structures of simple proteins is comparable to that of experimentally determined structures, providing broad possibilities for structure-based biological studies. Another critical question is whether and how multistate structures can be predicted from a given protein sequence. In this study, analysis of multiple two-state proteins demonstrated that deep learning-based contact map predictions contain structural information on both states, which suggests that it is probably appropriate to change the target of deep learning-based protein structure prediction from one specific structure to multiple likely structures. Furthermore, by combining deep learning- and physics-based computational methods, we developed a protocol for exploring alternative conformations from a known structure of a given protein, by which we successfully approached the holo-state conformation of a leucine-binding protein from its apo-state structure.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">li_exploring_2022</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Exploring the alternative conformation of a known protein structure based on contact map prediction}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{{AltConf}}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.biorxiv.org/content/10.1101/2022.06.07.495232v1}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/2022.06.07.495232}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2022-06-13}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{bioRxiv}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Li, Jiaxuan and Wang, Lei and Zhu, Zefeng and Song, Chen}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jun</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="wang_toward_2022" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Toward Atomistic Models of Intact SARS-CoV-2 via Martini Coarse-Grained Molecular Dynamics Simulations</div>
          <!-- Author -->
          <div class="author">Wang, Dali; Li, Jiaxuan; Wang, Lei; Cao, Yipeng; Li, Sai; and Song, Chen
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>bioRxiv</em> Jan 2022
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="http://biorxiv.org/content/early/2022/02/02/2022.01.31.478415.abstract" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The causative pathogen of Coronavirus disease 2019 (COVID-19), severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is an enveloped virus assembled by a lipid envelope, a positive-sense single-stranded RNA, and four structural proteins: the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins. Extensive experimental and computational studies have been carried out to reveal the architecture of the virus, but an intact and high-resolution structure of the virus is still desired. In this work, we construct atomistic models of the intact SARS-CoV-2 (without complete RNA) by fully employing the latest cryo-ET data, in combination with the available experimentally resolved protein structures, structure prediction and modeling methods, as well as coarse-grained molecular dynamics (MD) simulations. The atomistic model will not only provide a 3D structure for scientific demonstration and education, but also offer a framework for future exascale all-atom MD simulations to understand the dynamics of the intact virus and its mutations.Competing Interest StatementThe authors have declared no competing interest.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">MCP</abbr></div>

        <!-- Entry bib key -->
        <div id="wang_membrane_2022" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Membrane contact probability: an essential and predictive character for the structural and functional studies of membrane proteins</div>
          <!-- Author -->
          <div class="author">Wang, Lei; Zhang, Jiangguo; Wang, Dali; and Song, Chen
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>PLoS Computational Biology</em> Jan 2022
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.1371/journal.pcbi.1009972" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>One of the unique traits of membrane proteins is that a significant fraction of their hydrophobic amino acids is exposed to the hydrophobic core of lipid bilayers rather than being embedded in the protein interior, which is often not explicitly considered in the protein structure and function predictions. Here, we propose a characteristic and predictive quantity, the membrane contact probability (MCP), to describe the likelihood of the amino acids of a given sequence being in direct contact with the acyl chains of lipid molecules. We show that MCP is complementary to solvent accessibility in characterizing the outer surface of membrane proteins, and it can be predicted for any given sequence with a machine learning-based method by utilizing a training dataset extracted from MemProtMD, a database generated from molecular dynamics simulations for the membrane proteins with a known structure. As the first of many potential applications, we demonstrate that MCP can be used to systematically improve the prediction precision of the protein contact maps and structures.Competing Interest StatementThe authors have declared no competing interest.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">wang_membrane_2022</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Membrane contact probability: an essential and predictive character for the structural and functional studies of membrane proteins}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pcbi.1009972}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1371/journal.pcbi.1009972}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLoS Computational Biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{18}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e1009972}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Wang, Lei and Zhang, Jiangguo and Wang, Dali and Song, Chen}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{MCP}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="nv">true</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2021</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">NOMPC</abbr></div>

        <!-- Entry bib key -->
        <div id="wang_push--open_2021" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">The push-to-open mechanism of the tethered mechanosensitive ion channel NompC</div>
          <!-- Author -->
          <div class="author">Wang, Yang; Guo, Yifeng; Li, Guanluan; Liu, Chunhong; Wang, Lei; Zhang, Aihua; Yan, Zhiqiang; and Song, Chen
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>eLife</em> Jan 2021
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.7554/eLife.58388" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>NompC is a mechanosensitive ion channel responsible for the sensation of touch and balance in Drosophila melanogaster. Based on a resolved cryo-EM structure, we performed all-atom molecular dynamics simulations and electrophysiological experiments to study the atomistic details of NompC gating. Our results showed that NompC could be opened by compression of the intracellular ankyrin repeat domain but not by a stretch, and a number of hydrogen bonds along the force convey pathway are important for the mechanosensitivity. Under intracellular compression, the bundled ankyrin repeat region acts like a spring with a spring constant of ~13 pN nm−1 by transferring forces at a rate of ~1.8 nm ps−1. The linker helix region acts as a bridge between the ankyrin repeats and the transient receptor potential (TRP) domain, which passes on the pushing force to the TRP domain to undergo a clockwise rotation, resulting in the opening of the channel. This could be the universal gating mechanism of similar tethered mechanosensitive TRP channels, which enable cells to feel compression and shrinkage.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">wang_push--open_2021</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{The push-to-open mechanism of the tethered mechanosensitive ion channel {NompC}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{10}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2050-084X}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.7554/eLife.58388}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.7554/eLife.58388}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2021-06-09}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{eLife}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Wang, Yang and Guo, Yifeng and Li, Guanluan and Liu, Chunhong and Wang, Lei and Zhang, Aihua and Yan, Zhiqiang and Song, Chen}</span><span class="p">,</span>
  <span class="na">editor</span> <span class="p">=</span> <span class="s">{Delemotte, Lucie and Aldrich, Richard W and Martinac, Boris and Gaudet, Rachelle}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">note</span> <span class="p">=</span> <span class="s">{Publisher: eLife Sciences Publications, Ltd}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{electrophysiology, gating, mechanosensitive ion channel, molecular dynamics, NompC}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e58388}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{NOMPC}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">RyR</abbr></div>

        <!-- Entry bib key -->
        <div id="liu_atomistic_2021" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Atomistic Details of Charge/Space Competition in the Ca<sup>2+</sup> Selectivity of Ryanodine Receptors</div>
          <!-- Author -->
          <div class="author">Liu, Chunhong; Zhang, Aihua; Yan, Nieng; and Song, Chen
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>J. Phys. Chem. Lett.</em> Jan 2021
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.1021/acs.jpclett.1c00681" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Ryanodine receptors (RyRs) are ion channels responsible for the fast release of Ca2+ from the sarco/endoplasmic reticulum to the cytosol and show a selectivity of Ca2+ over monovalent cations. By utilizing a recently developed multisite Ca2+ model in molecular dynamic simulations, we show that multiple cations accumulate in the upper selectivity filter of RyRs, and the small size and high valence of Ca2+ make it preferable to K+ in competition for space in this confined region of negative electrostatic potential. The presence of Ca2+ in the upper selectivity filter significantly increases the energy barrier of K+ permeation, while the presence of K+ has little impact on the Ca2+ permeation. Our results provide the atomistic details of the charge/space competition mechanism for the ion selectivity of RyRs, which ensures the robustness of their Ca2+ release function. The mechanism could be utilized in protein- and nanoengineering for valence selectivity of ion species.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">liu_atomistic_2021</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Atomistic {Details} of {Charge}/{Space} {Competition} in the Ca&lt;sup&gt;2+&lt;/sup&gt; {Selectivity} of {Ryanodine} {Receptors}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1021/acs.jpclett.1c00681}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1021/acs.jpclett.1c00681}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2021-04-30}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{J. Phys. Chem. Lett.}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liu, Chunhong and Zhang, Aihua and Yan, Nieng and Song, Chen}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">note</span> <span class="p">=</span> <span class="s">{Publisher: American Chemical Society}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{4286--4291}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{RyR}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="nv">true</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">Orai</abbr></div>

        <!-- Entry bib key -->
        <div id="zhang_impact_2021" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">The Impact of Mutation L138F/L210F on the Orai Channel: A Molecular Dynamics Simulation Study</div>
          <!-- Author -->
          <div class="author">Zhang, Xiaoqian; Yu, Hua; Liu, Xiangdong; and Song, Chen
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Frontiers in Molecular Biosciences</em> Jan 2021
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.frontiersin.org/article/10.3389/fmolb.2021.755247" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The calcium release-activated calcium channel, composed of the Orai channel and the STIM protein, plays a crucial role in maintaining the Ca2+ concentration in cells. Previous studies showed that the L138F mutation in the human Orai1 creates a constitutively open channel independent of STIM, causing severe myopathy, but how the L138F mutation activates Orai1 is still unclear. Here, based on the crystal structure of Drosophila melanogaster Orai (dOrai), molecular dynamics simulations for the wild-type (WT) and the L210F (corresponding to L138F in the human Orai1) mutant were conducted to investigate their structural and dynamical properties. The results showed that the L210F dOrai mutant tends to have a more hydrated hydrophobic region (V174 to F171), as well as more dilated basic region (K163 to R155) and selectivity filter (E178). Sodium ions were located deeper in the mutant than in the wild-type. Further analysis revealed two local but essential conformational changes that may be the key to the activation. A rotation of F210, a previously unobserved feature, was found to result in the opening of the K163 gate through hydrophobic interactions. At the same time, a counter-clockwise rotation of F171 occurred more frequently in the mutant, resulting in a wider hydrophobic gate with more hydration. Ultimately, the opening of the two gates may facilitate the opening of the Orai channel independent of STIM.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang_impact_2021</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{Orai}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{The {Impact} of {Mutation} {L138F}/{L210F} on the {Orai} {Channel}: {A} {Molecular} {Dynamics} {Simulation} {Study}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2296-889X}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{The {Impact} of {Mutation} {L138F}/{L210F} on the {Orai} {Channel}}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/article/10.3389/fmolb.2021.755247}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmolb.2021.755247}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2021-11-02}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Molecular Biosciences}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Xiaoqian and Yu, Hua and Liu, Xiangdong and Song, Chen}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{755247}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="liu_putative_2021" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">A putative structural mechanism underlying the antithetic effect of homologous RND1 and RhoD GTPases in mammalian plexin regulation</div>
          <!-- Author -->
          <div class="author">Liu, Yanyan; Ke, Pu; Kuo, Yi-Chun; Wang, Yuxiao; Zhang, Xuewu; Song, Chen; and Shan, Yibing
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>eLife</em> Jan 2021
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.7554/eLife.64304" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Plexins are semaphorin receptors that play essential roles in mammalian neuronal axon guidance and in many other important mammalian biological processes. Plexin signaling depends on a semaphorin-induced dimerization mechanism and is modulated by small GTPases of the Rho family, of which RND1 serves as a plexin activator yet its close homolog RhoD an inhibitor. Using molecular dynamics (MD) simulations, we showed that RND1 reinforces the plexin dimerization interface, whereas RhoD destabilizes it due to their differential interaction with the cell membrane. Upon binding plexin at the Rho-GTPase-binding domain (RBD), RND1 and RhoD interact differently with the inner leaflet of the cell membrane and exert opposite effects on the dimerization interface via an allosteric network involving the RBD, RBD linkers, and a buttress segment adjacent to the dimerization interface. The differential membrane interaction is attributed to the fact that, unlike RND1, RhoD features a short C-terminal tail and a positively charged membrane interface.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">liu_putative_2021</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A putative structural mechanism underlying the antithetic effect of homologous {RND1} and {RhoD} {GTPases} in mammalian plexin regulation}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{10}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2050-084X}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.7554/eLife.64304}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.7554/eLife.64304}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2021-06-24}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{eLife}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Liu, Yanyan and Ke, Pu and Kuo, Yi-Chun and Wang, Yuxiao and Zhang, Xuewu and Song, Chen and Shan, Yibing}</span><span class="p">,</span>
  <span class="na">editor</span> <span class="p">=</span> <span class="s">{Frank, Aaron and Faraldo-Gómez, José D and Dickson, Alex and Chavent, Matthieu}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">note</span> <span class="p">=</span> <span class="s">{Publisher: eLife Sciences Publications, Ltd}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{membrane interaction, plexin, protein simulation, small GTPases}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e64304}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="li_synergetic_2021" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">A synergetic effect of BARD1 mutations on tumorigenesis</div>
          <!-- Author -->
          <div class="author">Li, Wenjing; Gu, Xiaoyang; Liu, Chunhong; Shi, Yanyan; Wang, Pan; Zhang, Na; Wu, Rui; Leng, Liang; Xie, Bingteng; Song, Chen; and Li, Mo
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Nature Communications</em> Jan 2021
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.nature.com/articles/s41467-021-21519-3" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>To date, a large number of mutations have been screened from breast and ovarian cancer patients. However, most of them are classified into benign or unidentified alterations due to their undetectable phenotypes. Whether and how they could cause tumors remains unknown, and this significantly limits diagnosis and therapy. Here, in a study of a family with hereditary breast and ovarian cancer, we find that two BARD1 mutations, P24S and R378S, simultaneously exist in cis in surviving cancer patients. Neither of the single mutations causes a functional change, but together they synergetically impair the DNA damage response and lead to tumors in vitro and in vivo. Thus, our report not only demonstrates that BARD1 defects account for tumorigenesis but also uncovers the potential risk of synergetic effects between the large number of cis mutations in individual genes in the human genome.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">li_synergetic_2021</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A synergetic effect of {BARD1} mutations on tumorigenesis}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{2021 The Author(s)}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2041-1723}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/s41467-021-21519-3}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41467-021-21519-3}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2021-03-08}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Communications}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Li, Wenjing and Gu, Xiaoyang and Liu, Chunhong and Shi, Yanyan and Wang, Pan and Zhang, Na and Wu, Rui and Leng, Liang and Xie, Bingteng and Song, Chen and Li, Mo}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">note</span> <span class="p">=</span> <span class="s">{Number: 1
  Publisher: Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1243}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2020</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">Ca<sup>2+</sup></abbr></div>

        <!-- Entry bib key -->
        <div id="zhang_ca2_2020" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">The Ca<sup>2+</sup> permeation mechanism of the ryanodine receptor revealed by a multi-site ion model</div>
          <!-- Author -->
          <div class="author">Zhang, Aihua; Yu, Hua; Liu, Chunhong; and Song, Chen
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Nature Communications</em> Jan 2020
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.nature.com/articles/s41467-020-14573-w" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Although the permeation mechanisms for K+ and Na+ channels have been extensively studied, the ion permeation mechanism through Ca2+ channels was largely unknown. Here the authors develop a multisite Ca2+ model that can be used in the framework of classical MD simulations to study Ca2+ in a quantitative manner, and use it to investigate the ion permeation mechanism of the ryanodine receptor 1.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang_ca2_2020</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{The Ca&lt;sup&gt;2+&lt;/sup&gt; permeation mechanism of the ryanodine receptor revealed by a multi-site ion model}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{2020 The Author(s)}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2041-1723}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/s41467-020-14573-w}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41467-020-14573-w}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2020-02-27}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Communications}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Aihua and Yu, Hua and Liu, Chunhong and Song, Chen}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{C922}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{Ca&lt;sup&gt;2+&lt;/sup&gt;}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="zhao_insights_2020" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Insights into the mechanism of membrane fusion induced by the plant defense element, plant-specific insert</div>
          <!-- Author -->
          <div class="author">Zhao, Xiaoli; Tian, Jenny (Jingxin); Yu, Hua; Bryksa, Brian C.; Dupuis, John H.; Ou, Xiuyuan; Qian, Zhaohui; Song, Chen; Wang, Shenlin; and Yada, Rickey Y.
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>J. Biol. Chem.</em> Jan 2020
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://www.jbc.org/content/295/43/14548" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>In plants, many natural defense mechanisms include cellular membrane fusion as a way to resist infection by external pathogens. Several plant proteins mediate membrane fusion, but the detailed mechanism by which they promote fusion is less clear. Understanding this process could provide valuable insights into these proteins’ physiological functions and guide bioengineering applications (i.e. the design of antimicrobial proteins). The plant-specific insert (PSI) from Solanum tuberosum can help reduce certain pathogen attack via membrane fusion. To gain new insights into the process of PSI-induced membrane fusion, a combined approach of NMR, FRET, and in silico studies was used. Our results indicate that (i) under acidic conditions, the PSI experiences a monomer-dimer equilibrium, and the dimeric PSI induces membrane fusion below a certain critical pH; (ii) after fusion, the PSI resides in a highly dehydrated environment with limited solvent accessibility, suggesting its capability in reducing repulsive dehydration forces between liposomes to facilitate fusion; and (iii) as shown by molecular dynamics simulations, the PSI dimer can bind stably to membrane surfaces and can bridge liposomes in close proximity, a critical step for the membrane fusion. In summary, this study provides new and unique insights into the mechanisms by which the PSI and similar proteins induce membrane fusion.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhao_insights_2020</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Insights into the mechanism of membrane fusion induced by the plant defense element, plant-specific insert}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{295}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0021-9258, 1083-351X}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://www.jbc.org/content/295/43/14548}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1074/jbc.RA120.014311}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{43}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2020-10-28}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{J. Biol. Chem.}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhao, Xiaoli and Tian, Jenny (Jingxin) and Yu, Hua and Bryksa, Brian C. and Dupuis, John H. and Ou, Xiuyuan and Qian, Zhaohui and Song, Chen and Wang, Shenlin and Yada, Rickey Y.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{32651232}</span><span class="p">,</span>
  <span class="na">note</span> <span class="p">=</span> <span class="s">{Publisher: American Society for Biochemistry and Molecular Biology}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{molecular dynamics, membrane fusion, membrane fusion mechanism, molecular dynamic simulation, NMR spectroscopy, nuclear magnetic resonance (NMR), plant defense, plant-specific insert, solid-state NMR}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{14548--14562}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="dupuis_role_2020" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">The role of disulfide bonds in a Solanum tuberosum saposin-like protein investigated using molecular dynamics</div>
          <!-- Author -->
          <div class="author">Dupuis, John H.; Wang, Shenlin; Song, Chen; and Yada, Rickey Y.
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>PLOS ONE</em> Jan 2020
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237884" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The Solanum tuberosum plant specific insert (StPSI) has a defensive role in potato plants, with the requirements of acidic pH and anionic lipids. The StPSI contains a set of three highly conserved disulfide bonds that bridge the protein’s helical domains. Removal of these bonds leads to enhanced membrane interactions. This work examined the effects of their sequential removal, both individually and in combination, using all-atom molecular dynamics to elucidate the role of disulfide linkages in maintaining overall protein tertiary structure. The tertiary structure was found to remain stable at both acidic (active) and neutral (inactive) pH despite the removal of disulfide linkages. The findings include how the dimer structure is stabilized and the impact on secondary structure on a residue-basis as a function of disulfide bond removal. The StPSI possesses an extensive network of inter-monomer hydrophobic interactions and intra-monomer hydrogen bonds, which is likely the key to the stability of the StPSI by stabilizing local secondary structure and the tertiary saposin-fold, leading to a robust association between monomers, regardless of the disulfide bond state. Removal of disulfide bonds did not significantly impact secondary structure, nor lead to quaternary structural changes. Instead, disulfide bond removal induces regions of amino acids with relatively higher or lower variation in secondary structure, relative to when all the disulfide bonds are intact. Although disulfide bonds are not required to preserve overall secondary structure, they may have an important role in maintaining a less plastic structure within plant cells in order to regulate membrane affinity or targeting.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dupuis_role_2020</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{The role of disulfide bonds in a {Solanum} tuberosum saposin-like protein investigated using molecular dynamics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{15}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1932-6203}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237884}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pone.0237884}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2020-08-28}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLOS ONE}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dupuis, John H. and Wang, Shenlin and Song, Chen and Yada, Rickey Y.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">note</span> <span class="p">=</span> <span class="s">{Publisher: Public Library of Science}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Molecular dynamics, Protein structure, Cysteine, Dimers, Disulfide bonds, Hydrogen bonding, Monomers, Salt bridges}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e0237884}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="wang_phosphorylation-dependent_2020" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Phosphorylation-dependent conformational changes of arrestin in the rhodopsin–arrestin complex</div>
          <!-- Author -->
          <div class="author">Wang, Dali; Liu, Xiangdong; Liu, Jianqiang; and Song, Chen
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Phys. Chem. Chem. Phys.</em> Jan 2020
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://pubs.rsc.org/en/content/articlelanding/2020/cp/d0cp00473a" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>G protein-coupled receptors (GPCRs) are membrane proteins that play critical roles in transmembrane signaling. Intracellular arrestin can form a complex with GPCRs to block G protein binding or mediate independent signaling pathways. It is known that different extracellular stimuli lead to the recruitment of different downstream effectors through arrestin. How this selective signaling is achieved is a fascinating but unresolved question. One hypothesis is that different stimuli can lead to different phosphorylation patterns in the C-terminus loop of GPCR (C-loop), and arrestin then adopts different conformations according to the phosphorylation pattern, and then arrestin in turn can recruit various downstream signaling molecules. In this study, we conducted atomistic molecular dynamics (MD) simulations to investigate whether the conformation of arrestin is related to the phosphorylation pattern of the GPCR C-loop in the GPCR–arrestin complex. Our results showed that arrestin undergoes a significant conformational change when binding to the GPCR C-loop, and its specific holo conformation seems to be phosphorylation-dependent. Further analysis of the pairwise forces between the phosphorylated residues of the C-loop and the adjacent residues of arrestin showed that these forces vary to a large degree, depending on the phosphorylation pattern of the C-loop, which might direct arrestin into distinct conformations and result in the selective binding of downstream signaling molecules. These results shed light on the C-loop phosphorylation pattern dependent signaling through the GPCR–arrestin pathway.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">wang_phosphorylation-dependent_2020</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Phosphorylation-dependent conformational changes of arrestin in the rhodopsin–arrestin complex}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{22}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1463-9084}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://pubs.rsc.org/en/content/articlelanding/2020/cp/d0cp00473a}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1039/D0CP00473A}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{17}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2020-05-06}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Phys. Chem. Chem. Phys.}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Wang, Dali and Liu, Xiangdong and Liu, Jianqiang and Song, Chen}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">note</span> <span class="p">=</span> <span class="s">{Publisher: The Royal Society of Chemistry}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{9330--9338}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2019</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="wu_chemistry_2019" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Chemistry of cation hydration and conduction in a skeletal muscle ryanodine receptor</div>
          <!-- Author -->
          <div class="author">Wu, Zhaolong; Liu, Congcong; Yu, Hua; Kang, Duan; Ma, Yinping; Li, Xuemei; Zhang, Lei; Fan, Chun; Li, Xin-Zheng; Song, Chen; Yin, Chang-Cheng; and Mao, Youdong
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>bioRxiv</em> Jan 2019
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.biorxiv.org/content/10.1101/732172v1" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>\textlessh3\textgreaterAbstract\textless/h3\textgreater \textlessp\textgreaterRyanodine receptors (RyRs) are Ca^2+-regulated Ca^2+ channels of 2.2-megadalton in muscles and neurons for calcium signaling. How Ca^2+ regulates ion conduction in the RyR channels remains elusive. We determined a 2.6-Å cryo-EM structure of rabbit skeletal muscle RyR1, and used multiscale dynamics simulations to elucidate cation interactions with RyR1. We investigated 21 potential cation-binding sites that may together rationalize biphasic Ca^2+ response of RyR1. The selectivity filter captures a cation hydration complex by hydrogen-bonding with both the inner and outer hydration shells of water molecules. Molecular dynamics simulations suggest that adjacent Ca^2+ ions moving in concert along ion-permeation pathway are separated by at least two cation-binding sites. Our analysis reveals that RyR1 has been evolved to favor its interactions with two hydration shells of cations.\textless/p\textgreater</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">wu_chemistry_2019</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Chemistry of cation hydration and conduction in a skeletal muscle ryanodine receptor}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission.}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.biorxiv.org/content/10.1101/732172v1}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1101/732172}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2019-11-11}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{bioRxiv}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Wu, Zhaolong and Liu, Congcong and Yu, Hua and Kang, Duan and Ma, Yinping and Li, Xuemei and Zhang, Lei and Fan, Chun and Li, Xin-Zheng and Song, Chen and Yin, Chang-Cheng and Mao, Youdong}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{732172}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">DCD</abbr></div>

        <!-- Entry bib key -->
        <div id="song_lipid_2019" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Lipid Bilayer Composition Influences the Activity of the Antimicrobial Peptide Dermcidin Channel</div>
          <!-- Author -->
          <div class="author">Song, Chen; Groot, Bert L.; and Sansom, Mark S. P.
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Biophysical Journal</em> Jan 2019
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.cell.com/biophysj/abstract/S0006-3495(19)30292-9" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Antimicrobial peptides (AMPs) carry great potential as new antibiotics against “superbugs.” Dermcidin (DCD), a broad-spectrum AMP in human sweat, has been recently crystallized in its oligomeric state and showed channel-like properties. In this work, we performed multiscale molecular dynamics simulations to study how the membrane composition influences the behavior of a transmembrane pore formed by the DCD oligomer in the hope of revealing the origin of the membrane selectivity of this AMP toward bacteria. Our results indicate that bilayers composed of various lipids (DMPC, DPPC, and DSPC) with different thicknesses result in different orientations of the DCD oligomer when embedded in lipid bilayers. The thicker the bilayer, the less tilted the channel. Cholesterol makes the bilayers more rigid and thicker, which also affects the orientation of the channel. Furthermore, we observed that the predicted conductance of the channel from computational electrophysiology simulations is related to its orientation in the lipid bilayer: the larger the tilt, the larger the conductance. Our results indicate that the membrane composition has a significant influence on the activity of the DCD channel, with thicker, cholesterol-rich membranes showing lower conductance than that of thinner membranes.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">song_lipid_2019</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Lipid {Bilayer} {Composition} {Influences} the {Activity} of the {Antimicrobial} {Peptide} {Dermcidin} {Channel}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{116}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0006-3495}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.cell.com/biophysj/abstract/S0006-3495(19)30292-9}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.bpj.2019.03.033}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2019-05-08}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Biophysical Journal}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Song, Chen and de Groot, Bert L. and Sansom, Mark S. P.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{31010668}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{DCD}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1658--1666}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="xu_deep_2019" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Deep learning for molecular generation</div>
          <!-- Author -->
          <div class="author">Xu, Youjun; Lin, Kangjie; Wang, Shiwei; Wang, Lei; Cai, Chenjing; Song, Chen; Lai, Luhua; and Pei, Jianfeng
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Future Medicinal Chemistry</em> Jan 2019
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.future-science.com/doi/full/10.4155/fmc-2018-0358" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>De novo drug design aims to generate novel chemical compounds with desirable chemical and pharmacological properties from scratch using computer-based methods. Recently, deep generative neural networks have become a very active research frontier in de novo drug discovery, both in theoretical and in experimental evidence, shedding light on a promising new direction of automatic molecular generation and optimization. In this review, we discussed recent development of deep learning models for molecular generation and summarized them as four different generative architectures with four different optimization strategies. We also discussed future directions of deep generative models for de novo drug design.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">xu_deep_2019</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Deep learning for molecular generation}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1756-8919}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.future-science.com/doi/full/10.4155/fmc-2018-0358}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.4155/fmc-2018-0358}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2019-04-14}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Future Medicinal Chemistry}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Xu, Youjun and Lin, Kangjie and Wang, Shiwei and Wang, Lei and Cai, Chenjing and Song, Chen and Lai, Luhua and Pei, Jianfeng}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{567--597}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="vestergaard_structure_2019" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Structure and Dynamics of Cinnamycin–Lipid Complexes: Mechanisms of Selectivity for Phosphatidylethanolamine Lipids</div>
          <!-- Author -->
          <div class="author">Vestergaard, Mikkel; Berglund, Nils Anton; Hsu, Pin-Chia; Song, Chen; Koldsø, Heidi; Schiøtt, Birgit; and Sansom, Mark S. P.
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>ACS Omega</em> Jan 2019
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.1021/acsomega.9b02949" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Cinnamycin is a lantibiotic peptide, which selectively binds to and permeabilizes membranes containing phosphatidylethanolamine (PE) lipids. As PE is a major component of many bacterial cell membranes, cinnamycin has considerable potential for destroying these. In this study, molecular dynamics simulations are used to elucidate the structure of a lipid–cinnamycin complex and the origin of selective lipid binding. The simulations reveal that cinnamycin selectively binds to PE by forming an extensive hydrogen-bonding network involving all three hydrogen atoms of the primary ammonium group of the PE head group. The substitution of a single hydrogen atom with a methyl group on the ammonium nitrogen destabilizes this hydrogen-bonding network. In addition to binding the primary ammonium group, cinnamycin also interacts with the phosphate group of the lipid through a previously uncharacterized phosphate-binding site formed by the backbone Phe10-Abu11-Phe12-Val13 moieties (Abu = 1-α-aminobutyric acid). In addition, hydroxylation of Asp15 at Cβ plays a role in selective binding of PE due to its tight interaction with the charged amine of the lipid head group. The simulations reveal that the position and orientation of the peptide in the membrane depend on the type of lipid to which it binds, suggesting a reason for why cinnamycin selectively permeabilizes PE-containing membranes.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">vestergaard_structure_2019</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Structure and {Dynamics} of {Cinnamycin}–{Lipid} {Complexes}: {Mechanisms} of {Selectivity} for {Phosphatidylethanolamine} {Lipids}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2470-1343}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Structure and {Dynamics} of {Cinnamycin}–{Lipid} {Complexes}}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1021/acsomega.9b02949}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1021/acsomega.9b02949}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{20}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2019-11-25}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{ACS Omega}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Vestergaard, Mikkel and Berglund, Nils Anton and Hsu, Pin-Chia and Song, Chen and Koldsø, Heidi and Schiøtt, Birgit and Sansom, Mark S. P.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{18889--18899}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2018</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">OSCA</abbr></div>

        <!-- Entry bib key -->
        <div id="zhang_structure_2018" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Structure of the mechanosensitive OSCA channels</div>
          <!-- Author -->
          <div class="author">Zhang, Mingfeng; Wang, Dali; Kang, Yunlu; Wu, Jing-Xiang; Yao, Fuqiang; Pan, Chengfang; Yan, Zhiqiang; Song, Chen; and Chen, Lei
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Nature Structural &amp; Molecular Biology</em> Jan 2018
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.nature.com/articles/s41594-018-0117-6" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Cryo-EM and electrophysiological studies of two mechanosensitive OSCA channels from Arabidopsis thaliana reveal their structural similarity to osmosensitive TMEM16 channels and suggest they are gated by force from lipid in response to osmotic stress.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang_structure_2018</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Structure of the mechanosensitive {OSCA} channels}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{25}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{2018 The Author(s), under exclusive licence to Springer Nature America, Inc.}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1545-9985}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/s41594-018-0117-6}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41594-018-0117-6}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2018-09-07}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Structural \&amp; Molecular Biology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Mingfeng and Wang, Dali and Kang, Yunlu and Wu, Jing-Xiang and Yao, Fuqiang and Pan, Chengfang and Yan, Zhiqiang and Song, Chen and Chen, Lei}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{850--858}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{OSCA}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="dupuis_ph_2018" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">pH dependent membrane binding of the Solanum tuberosum plant specific insert: An in silico study</div>
          <!-- Author -->
          <div class="author">Dupuis, John H.; Yu, Hua; Habibi, Mona; Peng, Xubiao; Plotkin, Steven S.; Wang, Shenlin; Song, Chen; and Yada, Rickey Y.
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Biochimica et Biophysica Acta (BBA) - Biomembranes</em> Jan 2018
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://www.sciencedirect.com/science/article/pii/S0005273618302943" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The Solanum tuberosum plant-specific insert (StPSI) has been shown to possess potent antimicrobial activity against both human and plant pathogens. Furthermore, in vitro, the StPSI is capable of fusing phospholipid vesicles, provided the conditions of net anionic vesicle charge and acidic pH are met. Constant pH replica-exchange simulations indicate several acidic residues on the dimer have highly perturbed pKas (\textless3.0; E15, D28, E85 &amp; E100) due to involvement in salt bridges. After setting the pH of the system to either 3.0 or 7.4, all-atom simulations provided details of the effect of pH on secondary structural elements, particularly in the previously unresolved crystallographic structure of the loop section. Coarse-grained dimer-bilayer simulations demonstrated that at pH 7.4, the dimer had no affinity for neutral or anionic membranes over the course of 1 μs simulations. Conversely, at pH 3.0 two binding modes were observed. Mode 1 is mediated primarily via strong N-terminal interactions on one monomer only, whereas in mode 2, N- and C-terminal residues of one monomer and numerous polar and basic residues on the second monomer, particularly in the third helix, participate in membrane interactions. Mode 2 was accompanied by re-orientation of the dimer to a more vertical position with respect to helices 1 and 4, positioning the dimer for membrane interactions. These results offer the first examination at near-atomic resolution of residues mediating the StPSI-membrane interactions, and allow for the postulation of a possible fusion mechanism.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dupuis_ph_2018</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{pH} dependent membrane binding of the {Solanum} tuberosum plant specific insert: {An} in silico study}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{1860}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0005-2736}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{{pH} dependent membrane binding of the {Solanum} tuberosum plant specific insert}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://www.sciencedirect.com/science/article/pii/S0005273618302943}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.bbamem.2018.10.001}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2018-10-10}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Biochimica et Biophysica Acta (BBA) - Biomembranes}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dupuis, John H. and Yu, Hua and Habibi, Mona and Peng, Xubiao and Plotkin, Steven S. and Wang, Shenlin and Song, Chen and Yada, Rickey Y.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Molecular dynamics, Membrane interactions, Protein-membrane interactions, Saposin-like protein}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2608--2618}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2017</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">GA</abbr></div>

        <!-- Entry bib key -->
        <div id="wang_orientation_2017" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">The orientation and stability of the GPCR-Arrestin complex in a lipid bilayer</div>
          <!-- Author -->
          <div class="author">Wang, Dali; Yu, Hua; Liu, Xiangdong; Liu, Jianqiang; and Song, Chen
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Scientific Reports</em> Jan 2017
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.nature.com/articles/s41598-017-17243-y" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>G protein-coupled receptors (GPCRs) constitute a large family of membrane proteins that plays a key role in transmembrane signal transduction and draw wide attention since it was discovered. Arrestin is a small family of proteins which can bind to GPCRs, block G protein interactions and redirect signaling to G-protein-independent pathways. The detailed mechanism of how arrestin interacts with GPCR remains elusive. Here, we conducted molecular dynamics simulations with coarse-grained (CG) and all-atom (AA) models to study the complex structure formed by arrestin and rhodopsin, a prototypical GPCR, in a POPC bilayer. Our results indicate that the formation of the complex has a significant impact on arrestin which is tightly anchored onto the bilayer surface, while has a minor effect on the orientation of rhodopsin in the lipid bilayer. The formation of the complex induces an internal change of conformation and flexibility in both rhodopsin and arrestin, mainly at the binding interface. Further investigation on the interaction interface identified the hydrogen bond network, especially the long-lived hydrogen bonds, and the key residues at the contact interface, which are responsible for stabilizing the complex. These results help us to better understand how rhodopsin interacts with arrestin on membranes, and thereby shed lights on arrestin-mediated signal transduction through GPCRs.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">wang_orientation_2017</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{GA}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{The orientation and stability of the {GPCR}-{Arrestin} complex in a lipid bilayer}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{2017 The Author(s)}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2045-2322}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/s41598-017-17243-y}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41598-017-17243-y}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2018-08-29}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Scientific Reports}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Wang, Dali and Yu, Hua and Liu, Xiangdong and Liu, Jianqiang and Song, Chen}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{16985}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="trick_voltage_2017" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Voltage Gating of a Biomimetic Nanopore: Electrowetting of a Hydrophobic Barrier</div>
          <!-- Author -->
          <div class="author">Trick, Jemma L.; Song, Chen; Wallace, E. Jayne; and Sansom, Mark S. P.
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>ACS Nano</em> Jan 2017
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.1021/acsnano.6b07865" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>It is desirable that nanopores that are components of biosensors are gated, i.e., capable of controllable switching between closed (impermeable) and open (permeable) states. A central hydrophobic barrier within a nanopore may act as a voltage-dependent gate via electrowetting, i.e., changes in nanopore surface wettability by application of an electric field. We use “computational electrophysiology” simulations to demonstrate and characterize electrowetting of a biomimetic nanopore containing a hydrophobic gate. We show that a hydrophobic gate in a model β-barrel nanopore can be functionally opened by electrowetting at voltages that do not electroporate lipid bilayers. During the process of electrowetting, voltage-induced alignment of water dipoles occurs within the hydrophobic gate region of the nanopore, with water entry preceding permeation of ions through the opened nanopore. When the ionic imbalance that generates a transbilayer potential is dissipated, water is expelled from the hydrophobic gate and the nanopore recloses. The open nanopore formed by electrowetting of a “featureless” β-barrel is anionic selective due to the transmembrane dipole potential resulting from binding of Na+ ions to the headgroup regions of the surrounding lipid bilayer. Thus, hydrophobic barriers can provide voltage-dependent gates in designed biomimetic nanopores. This extends our understanding of hydrophobic gating in synthetic and biological nanopores, providing a framework for the design of functional nanopores with tailored gating functionality.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">trick_voltage_2017</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Voltage {Gating} of a {Biomimetic} {Nanopore}: {Electrowetting} of a {Hydrophobic} {Barrier}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1936-0851}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Voltage {Gating} of a {Biomimetic} {Nanopore}}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1021/acsnano.6b07865}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1021/acsnano.6b07865}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2018-08-29}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{ACS Nano}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Trick, Jemma L. and Song, Chen and Wallace, E. Jayne and Sansom, Mark S. P.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1840--1847}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2016</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="kutzner_insights_2016" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Insights into the function of ion channels by computational electrophysiology simulations</div>
          <!-- Author -->
          <div class="author">Kutzner, Carsten; Köpfer, David A.; Machtens, Jan-Philipp; Groot, Bert L.; Song, Chen; and Zachariae, Ulrich
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Biochimica et Biophysica Acta (BBA) - Biomembranes</em> Jan 2016
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://www.sciencedirect.com/science/article/pii/S0005273616300360" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Ion channels are of universal importance for all cell types and play key roles in cellular physiology and pathology. Increased insight into their functional mechanisms is crucial to enable drug design on this important class of membrane proteins, and to enhance our understanding of some of the fundamental features of cells. This review presents the concepts behind the recently developed simulation protocol Computational Electrophysiology (CompEL), which facilitates the atomistic simulation of ion channels in action. In addition, the review provides guidelines for its application in conjunction with the molecular dynamics software package GROMACS. We first lay out the rationale for designing CompEL as a method that models the driving force for ion permeation through channels the way it is established in cells, i.e., by electrochemical ion gradients across the membrane. This is followed by an outline of its implementation and a description of key settings and parameters helpful to users wishing to set up and conduct such simulations. In recent years, key mechanistic and biophysical insights have been obtained by employing the CompEL protocol to address a wide range of questions on ion channels and permeation. We summarize these recent findings on membrane proteins, which span a spectrum from highly ion-selective, narrow channels to wide diffusion pores. Finally we discuss the future potential of CompEL in light of its limitations and strengths. This article is part of a Special Issue entitled: Membrane Proteins edited by J.C. Gumbart and Sergei Noskov.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">kutzner_insights_2016</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">series</span> <span class="p">=</span> <span class="s">{New approaches for bridging computation and experiment on membrane proteins}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Insights into the function of ion channels by computational electrophysiology simulations}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{1858}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0005-2736}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://www.sciencedirect.com/science/article/pii/S0005273616300360}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.bbamem.2016.02.006}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7, Part B}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2018-08-29}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Biochimica et Biophysica Acta (BBA) - Biomembranes}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kutzner, Carsten and Köpfer, David A. and Machtens, Jan-Philipp and de Groot, Bert L. and Song, Chen and Zachariae, Ulrich}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2016}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Molecular dynamics, Electrophysiology, Selectivity, Ion channels, Conductance, GROMACS}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1741--1752}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>

  <h2 class="year">2015</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="yang_phospho-selective_2015" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Phospho-selective mechanisms of arrestin conformations and functions revealed by unnatural amino acid incorporation and <sup>19</sup>F-NMR</div>
          <!-- Author -->
          <div class="author">Yang, Fan; Yu, Xiao; Liu, Chuan; Qu, Chang-Xiu; Gong, Zheng; Liu, Hong-Da; Li, Fa-Hui; Wang, Hong-Mei; He, Dong-Fang; Yi, Fan; Song, Chen; Tian, Chang-Lin; Xiao, Kun-Hong; Wang, Jiang-Yun; and Sun, Jin-Peng
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Nature Communications</em> Jan 2015
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.nature.com/articles/ncomms9202" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Specific arrestin conformations are coupled to distinct downstream effectors, which underlie the functions of many G-protein-coupled receptors (GPCRs). Here, using unnatural amino acid incorporation and fluorine-19 nuclear magnetic resonance (19F-NMR) spectroscopy, we demonstrate that distinct receptor phospho-barcodes are translated to specific β-arrestin-1 conformations and direct selective signalling. With its phosphate-binding concave surface, β-arrestin-1 ‘reads’ the message in the receptor phospho-C-tails and distinct phospho-interaction patterns are revealed by 19F-NMR. Whereas all functional phosphopeptides interact with a common phosphate binding site and induce the movements of finger and middle loops, different phospho-interaction patterns induce distinct structural states of β-arrestin-1 that are coupled to distinct arrestin functions. Only clathrin recognizes and stabilizes GRK2-specific β-arrestin-1 conformations. The identified receptor-phospho-selective mechanism for arrestin conformation and the spacing of the multiple phosphate-binding sites in the arrestin enable arrestin to recognize plethora phosphorylation states of numerous GPCRs, contributing to the functional diversity of receptors.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">yang_phospho-selective_2015</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Phospho-selective mechanisms of arrestin conformations and functions revealed by unnatural amino acid incorporation and &lt;sup&gt;{19}&lt;/sup&gt;{F}-{NMR}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{2015 Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2041-1723}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/ncomms9202}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/ncomms9202}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2018-06-29}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Communications}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Yang, Fan and Yu, Xiao and Liu, Chuan and Qu, Chang-Xiu and Gong, Zheng and Liu, Hong-Da and Li, Fa-Hui and Wang, Hong-Mei and He, Dong-Fang and Yi, Fan and Song, Chen and Tian, Chang-Lin and Xiao, Kun-Hong and Wang, Jiang-Yun and Sun, Jin-Peng}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{8202}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>

  <h2 class="year">2014</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">KKK</abbr></div>

        <!-- Entry bib key -->
        <div id="Kopfer2014" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Ion permeation in K<sup>+</sup> channels occurs by direct Coulomb knock-on</div>
          <!-- Author -->
          <div class="author">Köpfer, David A.; Song, Chen; Gruene, Tim; Sheldrick, George M.; Zachariae, Ulrich; and Groot, Bert L.
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Science</em> Jan 2014
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://www.sciencemag.org/cgi/doi/10.1126/science.1254840" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Potassium channels selectively conduct K+ ions across cellular membranes with extraordinary efficiency. Their selectivity filter exhibits four binding sites with approximately equal electron density in crystal structures with high K+ concentrations, previously thought to reflect a superposition of alternating ion- and water-occupied states. Consequently, cotranslocation of ions with water has become a widely accepted ion conduction mechanism for potassium channels. By analyzing more than 1300 permeation events from molecular dynamics simulations at physiological voltages, we observed instead that permeation occurs via ion-ion contacts between neighboring K+ ions. Coulomb repulsion between adjacent ions is found to be the key to high-efficiency K+ conduction. Crystallographic data are consistent with directly neighboring K+ ions in the selectivity filter, and our model offers an intuitive explanation for the high throughput rates of K+ channels. Simulation shows that ions crossing a potassium channel are in direct contact with one another and repel each other through. [Also see Perspective by Hummer] Potassium channels play a key role in regulating a cell’s membrane potential, which in turn affects diverse processes. The channels contain four potassium binding sites that are thought to be alternately occupied by potassium and water. Starting from high-resolution crystal structures, Köpfer et al. simulated over a thousand potassium ions crossing the channel (see the Perspective by Hummer). They found that ions are in direct contact rather than being separated by water as previously assumed. It seems that repulsion between the ions is the key to their efficient movement through the channel. Science, this issue p. 352; see also p. 303</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Kopfer2014</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Köpfer, David A. and Song, Chen and Gruene, Tim and Sheldrick, George M. and Zachariae, Ulrich and de Groot, Bert L.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Ion permeation in K&lt;sup&gt;+&lt;/sup&gt; channels occurs by direct Coulomb knock-on}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Science}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://www.sciencemag.org/cgi/doi/10.1126/science.1254840}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="nv">true</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{KKK}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{346}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6207}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{352-355}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1126/science.1254840}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://www.science.org/doi/pdf/10.1126/science.1254840}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>

  <h2 class="year">2013</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">DCD</abbr></div>

        <!-- Entry bib key -->
        <div id="song2013" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Crystal structure and functional mechanism of a human antimicrobial membrane channel</div>
          <!-- Author -->
          <div class="author">Song, Chen; Weichbrodt, Conrad; Salnikov, Evgeniy S.; Dynowski, Marek; Forsberg, Björn O.; Bechinger, Burkhard; Steinem, Claudia; Groot, Bert L.; Zachariae, Ulrich; and Zeth, Kornelius
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Proc. Natl. Acad. Sci. U.S.A.</em> Jan 2013
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.pnas.org/doi/abs/10.1073/pnas.1214739110" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p> Multicellular organisms fight bacterial and fungal infections by producing peptide-derived broad-spectrum antibiotics. These host-defense peptides compromise the integrity of microbial cell membranes and thus evade pathways by which bacteria develop rapid antibiotic resistance. Although more than 1,700 host-defense peptides have been identified, the structural and mechanistic basis of their action remains speculative. This impedes the desired rational development of these agents into next-generation antibiotics. We present the X-ray crystal structure as well as solid-state NMR spectroscopy, electrophysiology, and MD simulations of human dermcidin in membranes that reveal the antibiotic mechanism of this major human antimicrobial, found to suppress Staphylococcus aureus growth on the epidermal surface. Dermcidin forms an architecture of high-conductance transmembrane channels, composed of zinc-connected trimers of antiparallel helix pairs. Molecular dynamics simulations elucidate the unusual membrane permeation pathway for ions and show adjustment of the pore to various membranes. Our study unravels the comprehensive mechanism for the membrane-disruptive action of this mammalian host-defense peptide at atomistic level. The results may form a foundation for the structure-based design of peptide antibiotics. </p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">song2013</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Song, Chen and Weichbrodt, Conrad and Salnikov, Evgeniy S. and Dynowski, Marek and Forsberg, Björn O. and Bechinger, Burkhard and Steinem, Claudia and de Groot, Bert L. and Zachariae, Ulrich and Zeth, Kornelius}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Crystal structure and functional mechanism of a human antimicrobial membrane channel}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proc. Natl. Acad. Sci. U.S.A.}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{110}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{4586-4591}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2013}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1073/pnas.1214739110}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.pnas.org/doi/abs/10.1073/pnas.1214739110}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="nv">true</span><span class="p">,</span>
  <span class="na">abbr</span> <span class="p">=</span> <span class="s">{DCD}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>

  <h2 class="year">2011</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="song_testing_2011" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Testing the applicability of Nernst-Planck theory in ion channels: comparisons with Brownian dynamics simulations.</div>
          <!-- Author -->
          <div class="author">Song, Chen; and Corry, Ben
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>PLoS one</em> Jan 2011
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/21731672" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The macroscopic Nernst-Planck (NP) theory has often been used for predicting ion channel currents in recent years, but the validity of this theory at the microscopic scale has not been tested. In this study we systematically tested the ability of the NP theory to accurately predict channel currents by combining and comparing the results with those of Brownian dynamics (BD) simulations. To thoroughly test the theory in a range of situations, calculations were made in a series of simplified cylindrical channels with radii ranging from 3 to 15 Å, in a more complex ’catenary’ channel, and in a realistic model of the mechanosensitive channel MscS. The extensive tests indicate that the NP equation is applicable in narrow ion channels provided that accurate concentrations and potentials can be input as the currents obtained from the combination of BD and NP match well with those obtained directly from BD simulations, although some discrepancies are seen when the ion concentrations are not radially uniform. This finding opens a door to utilising the results of microscopic simulations in continuum theory, something that is likely to be useful in the investigation of a range of biophysical and nano-scale applications and should stimulate further studies in this direction.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">song_testing_2011</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Testing the applicability of {Nernst}-{Planck} theory in ion channels: comparisons with {Brownian} dynamics simulations.}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://www.ncbi.nlm.nih.gov/pubmed/21731672}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pone.0021204}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLoS one}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Song, Chen and Corry, Ben}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2011}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e21204--e21204}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>

  <h2 class="year">2010</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="song_ion_2010" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Ion Conduction in Ligand-Gated Ion Channels: Brownian Dynamics Studies of Four Recent Crystal Structures</div>
          <!-- Author -->
          <div class="author">Song, Chen; and Corry, Ben
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Biophysical Journal</em> Jan 2010
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://www.cell.com/biophysj/abstract/S0006-3495(09)01674-9" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Four x-ray crystal structures of prokaryotic homologs of ligand-gated ion channels have recently been determined: ELIC from Erwinia chrysanthemi, two structures of a proton-activated channel from Gloebacter violaceus (GLIC1 and GLIC2) and that of the E221A mutant (GLIC1M). The availability of numerous structures of channels in this family allows for aspects of channel gating and ion conduction to be examined. Here, we determine the likely conduction states of the four structures as well as IV curves, ion selectivity, and steps involved in ion permeation by performing extensive Brownian dynamics simulations. Our results show that the ELIC structure is indeed nonconductive, but that GLIC1 and GLIC1M are both conductive of ions with properties different from those seen in experimental studies of the channel. GLIC2 appears to reflect an open state of the channel with a predicted conductance of 10.8–12.4 pS in 140 mM NaCl solution, which is comparable to the experimental value 8 ± 2 pS. The extracellular domain of the channel is shown to have an important influence on the channel current, but a less significant role in ion selectivity.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">song_ion_2010</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Ion {Conduction} in {Ligand}-{Gated} {Ion} {Channels}: {Brownian} {Dynamics} {Studies} of {Four} {Recent} {Crystal} {Structures}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{98}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0006-3495}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Ion {Conduction} in {Ligand}-{Gated} {Ion} {Channels}}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.cell.com/biophysj/abstract/S0006-3495(09)01674-9}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.bpj.2009.10.032}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2018-08-29}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Biophysical Journal}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Song, Chen and Corry, Ben}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2010}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{20141753}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{404--411}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>

  <h2 class="year">2009</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="song_role_2009" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Role of Acetylcholine Receptor Domains in Ion Selectivity</div>
          <!-- Author -->
          <div class="author">Song, Chen; and Corry, Ben
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>BBA-Biomembranes</em> Jan 2009
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/19397891" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The nicotinic acetylcholine receptor (nAChR) is a ligand gated ion channel protein, composed of three domains: a transmembrane domain (TM-domain), extracellular domain (EC-domain), and intracellular domain (IC-domain). Due to its biological importance, much experimental and theoretical research has been carried out to explore its mechanisms of gating and selectivity, but there are still many unresolved issues, especially on the ion selectivity. Moreover, most of the previous theoretical work has concentrated on the TM-domain or EC-domain of nAChR, which may be insufficient to understand the entire structure-function relation. In this work, we perform molecular dynamics, Brownian dynamics simulations and continuum electrostatic calculations to investigate the role of different nAChR domains in ion conduction and selectivity. The results show that although both the EC and IC domains contain strong negative charges that create large cation concentrations at either end of the pore, this alone is not sufficient to create the observed cation selectivity and may play a greater role in determining the channel conductance. The presence of cations in the wide regions of the pore can screen out the protein charge allowing anions to enter, meaning that local regions of the TM-domain are most likely responsible for discriminating between ions. These new results complement our understanding about the ion conduction and selectivity mechanism of nAChR.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">song_role_2009</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Role of {Acetylcholine} {Receptor} {Domains} in {Ion} {Selectivity}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{1788}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://www.ncbi.nlm.nih.gov/pubmed/19397891}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.bbamem.2009.04.015}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{BBA-Biomembranes}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Song, Chen and Corry, Ben}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2009}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Protein Structure, Amino Acid Sequence, Molecular Sequence Data, Cations, Static Electricity, Mutation, Tertiary, Cations: chemistry, Nicotinic, Nicotinic: chemistry, Nicotinic: genetics, Receptors}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1466--1473}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="song_intrinsic_2009" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Intrinsic Ion Selectivity of Narrow Hydrophobic Pores</div>
          <!-- Author -->
          <div class="author">Song, Chen; and Corry, Ben
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>J. Phys. Chem. B</em> Jan 2009
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.1021/jp810102u" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>We show that narrow hydrophobic pores have an intrinsic ion selectivity by using single-walled carbon nanotube membranes as a model. We examined pores of radius 3.4−6.1 Å, and conducted molecular dynamics simulations to show that Na+, K+, and Cl− face different free energy barriers when entering hydrophobic pores. Most of the differences result from the different dehydration energies of the ions; however, changes in the solvation shell structure in the confined nanotube interior and van der Waals interactions in the small tubes can both play a role. Molecular dynamics simulations conducted under hydrostatic pressure show that carbon nanotube membranes can act as ion sieves, with the pore radius and pressure determining which ions will permeate through the membrane. This work suggests that the intrinsic ion selectivity of biological pores of differing radii might also play a role in determining their selectivity, in addition to the more common explanations based on electrostatic effects. In addition, “hydrophobic gating” can arise in continuous water-filled pores.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">song_intrinsic_2009</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Intrinsic {Ion} {Selectivity} of {Narrow} {Hydrophobic} {Pores}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{113}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1520-6106}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1021/jp810102u}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1021/jp810102u}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{21}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2018-09-06}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{J. Phys. Chem. B}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Song, Chen and Corry, Ben}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2009}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{7642--7649}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="song_computational_2009" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Computational study of the transmembrane domain of the acetylcholine receptor</div>
          <!-- Author -->
          <div class="author">Song, Chen; and Corry, Ben
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Eur Biophys J</em> Jan 2009
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.1007/s00249-009-0476-3" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The nicotinic acetylcholine receptor (nAChR) is a ligand-gated ion channel protein whose transmembrane domain (TM-domain) is believed to be responsible for channel gating via a hydrophobic effect. In this work, we perform molecular dynamics and Brownian dynamics simulations to investigate the effect of transmembrane potential on the conformation and water occupancy of TM-domain, and the resulting ion permeation events. The results show that the behavior of the hydrophobic gate is voltage-dependent. Large hyperpolarized membrane potential can change the conformation of TM-domain and water occupancy in this region, which may enable ion conduction. An electrostatic gating mechanism is also proposed from our simulations, which seems to play a role in addition to the well-known hydrophobic effect.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">song_computational_2009</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Computational study of the transmembrane domain of the acetylcholine receptor}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{38}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1432-1017}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1007/s00249-009-0476-3}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s00249-009-0476-3}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2018-08-29}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Eur Biophys J}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Song, Chen and Corry, Ben}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2009}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Molecular dynamics, Acetylcholine receptor, Brownian dynamics, Gating, Ion channel, Membrane potential}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{961--970}</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>


</div>

          </article>

        </div>

    </div>

    <!-- Footer -->    <footer class="sticky-bottom mt-5">
      <div class="container">
        © Copyright 2023 Chen  SONG. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="noopener noreferrer">GitHub Pages</a>.

      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
  <script src="/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script src="/assets/js/common.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
  </body>
</html>

